backend/business-partner-agent/src/main/resources/application.yml

```diff
@@ -142,6 +142,18 @@ bpa:
   i18n:
     locale: ${BPA_I18N_LOCALE:en}
     fallbackLocale: ${BPA_I18N_FALLBACK_LOCALE:en}
+  activityLog:
```

Comment:
Sorry for not being precise enough. What I meant was that whether a task should become active or not should depend on the state of the auto flags set on aca-py. 

I would propose to add a subclass called AcaPyConfig to the RuntimeConfig and initialize it in the [onApplicationEvent](https://github.com/hyperledger-labs/business-partner-agent/blob/9141903fbe47961902141601ab3e2efad271aba7/backend/business-partner-agent/src/main/java/org/hyperledger/bpa/config/RuntimeConfig.java#L90) in there you have access to all the values mainly:

```
   "debug.auto_respond_credential_proposal": true,
    "debug.auto_respond_credential_offer": true,
    "debug.auto_respond_credential_request": true,
    "debug.auto_respond_presentation_proposal": true,
    "debug.auto_respond_presentation_request": true,
    "debug.auto_store_credential": true,
    "debug.auto_verify_presentation": true,
    "debug.auto_accept_invites": true,
    "debug.auto_accept_requests": true,
    "debug.auto_respond_messages": true,
```

If an auto flag is set to false or not set at all then I would expect the task to be active, because otherwise the user will see events and tasks where there are none. Does this make sense?

Reply:
OK, so you do NOT want to configure which events would be considered COMPLETE/etc, you just want the acapy flags read through RuntimeConfig?

The events that would be for TASK, ACTIVITY LOG and COMPLETED can be statics in Java, and put the debug flags into RuntimeConfig.AcaPyConfig.